components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -96,6 +96,8 @@ class GeckoEngineSession(
     internal var currentUrl: String? = null
     internal var lastLoadRequestUri: String? = null
     internal var pageLoadingUrl: String? = null
+    internal var appRedirectUrl: String? = null
+    internal var isLastAppredirectedUri: Boolean = false
```

Comment:
Do we need both of these? couldn't `appRedirectUrl != null` replaces the check `isLastAppredirectUri == true?

Reply:
Sure. We can use a single variable